classes/Mail.php

```diff
@@ -656,7 +656,7 @@ function ($carry, $item) {
                 'Swift Error: ' . $e->getMessage(),
                 3,
                 null,
-                'Swift_Message'
+                'SwiftMessage'
```

Comment:
Why did you change this?

Reply:
I changed it, because all other `$objectType`s are in camelcase.

But since I didn't change the occurrence of `Swift_Message` in `PrestashopLogger`, it's bad.

Reply:
In fact, I commented before on the reason why I changed this:
https://github.com/PrestaShop/PrestaShop/pull/24707#issuecomment-857672010

`Swift_Message` does not validate with the `isName` rule, but it has to.
The right correction is to change `Swift_Message` to  `SwiftMessage` everywhere.